package.json

```diff
@@ -13,7 +13,7 @@
     "nan": "^2.4.0"
   },
   "devDependencies": {
-    "tree-sitter-cli": "^0.5.1",
```

Comment:
@maxbrunsfeld I thought now might be a good chance to update `tree-sitter-cli`, too, and update the test files from `grammar_test` to `corpus`, to match the other tree-sitter repos. I can also move these out into a separate PR if you'd prefer.